### PR TITLE
Comments & `@AnnotatedFor` from nullness-java-files 

### PR DESCRIPTION
**Pull request by the1derer**
~Don't merge before merging #19~

### CODE REVIEW

**Comment by mernst** [comment]
In your clone of this branch, please follow the instructions about "Pull request merge conflicts" at https://github.com/typetools/jdk/blob/master/README.  I think that will make the CI build pass.

**Comment by the1derer** [comment]
@mernst Ready for merging.

**Comment by mernst** [comment]
The CI build is failing.  Please correct that before assigning to me.  Thanks.

**Comment by the1derer** [comment]
@mernst 
Passed all tests and ready to be merged.

File: src/java.base/share/classes/java/io/BufferedWriter.java [hunk 1]
```diff
@@ -29,6 +29,8 @@
 import org.checkerframework.checker.index.qual.NonNegative;
 import org.checkerframework.checker.index.qual.Positive;
 
+import org.checkerframework.framework.qual.AnnotatedFor;
+
```
Comment (mernst) [inline src/java.base/share/classes/java/io/BufferedWriter.java]: Please try not to add extra whitespace.

File: src/java.base/share/classes/java/io/BufferedWriter.java [hunk 2]
```diff
@@ -29,6 +29,8 @@
 import org.checkerframework.checker.index.qual.NonNegative;
 import org.checkerframework.checker.index.qual.Positive;
```
Comment (mernst) [inline src/java.base/share/classes/java/io/BufferedWriter.java]: Please put all the Checker Framework annotations in a single block of `import` statements, without blank lines within them.

File: src/java.base/share/classes/java/io/FileOutputStream.java [hunk 3]
```diff
@@ -25,6 +25,7 @@
 
 package java.io;
 
+import org.checkerframework.framework.qual.AnnotatedFor;
```
Comment (mernst) [inline src/java.base/share/classes/java/io/FileOutputStream.java]: Please put import statements in lexicographic order.  It makes merges easier.

**Comment by the1derer** [comment]
> I noticed some issues with the import statements. I will resolve them this time around, then proceed with my review.

These(import statement other than `@AnnotatedFor` and `@CFComment`) are added when I pulled the latest changes from `typetools/jdk:comments-from-java-files` which introduced all other unwanted whitespace changes and disturbed the lexicographic order.
I will surely try to avoid this in the next PRs.

Thanks.

**Pull request merged**